tests/AppTest.php

```diff
@@ -219,8 +218,10 @@ public function testHttpFlowEventNotInstanceOfEventInterface()
         $response = $app->run($request, $response);
     }
 
-    public function testRouteInfoNotInstanceOfRouteInfoInterface()
+    public function testDispatcherNotInstanceOfPennyDispatcher()
     {
+        $this->setExpectedException('RuntimeException', 'Dispatcher must be an instance of Penny\Dispatcher class');
```

Comment:
this uses non dispatcher class, uses closure : 
```php
$dispatcher = function() use ($request) {
      return 'callback';
};
```
that's why the response *was* return null, and silent.